tests/unit/broccoli/default-packager/tests-test.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const p = require('ember-cli-preprocess-registry/preprocessors');
+const co = require('co');
+const expect = require('chai').expect;
+const DefaultPackager = require('../../../../lib/broccoli/default-packager');
+const broccoliTestHelper = require('broccoli-test-helper');
+
+const buildOutput = broccoliTestHelper.buildOutput;
+const createTempDir = broccoliTestHelper.createTempDir;
+
+const DEFAULT_DEPENDENCIES = {
```

Comment:
Would it make sense to read this from something like https://github.com/ember-cli/ember-cli/blob/master/tests/fixtures/app/npm/package.json to prevent more duplication?

Reply:
I thought about that but I think we should 🔥 those fixture files so decided to create an object instead. Also, packager will be extracted into a separate package.

Reply:
I would use it until you extract. So someone doing a PR to update a package in the blueprint, not knowing about this test, will find all/replace the fixtures and wonder if they should touch this test or not.

Also these fixtures are needed to prevent regressing template conditional content (we regressed often without them).

Reply:
To be clear, I added this object _only_ to be able to use `p.defaultRegistry()` b/c I didn't want to set up a fake one; it will not break if someone will do find all/replace and has nothing to do with blueprint's dependencies. Having said that, if you feel strongly, I can change.

Reply:
What dependencies does defaultRegistry require? Could we give it just the deps it needs to succeed?